setup.py

```diff
@@ -33,7 +33,7 @@
     license='Apache License 2.0',
     install_requires=[
         'tensorflow>=2.0.0-beta1',
-        'numpy',
+        'numpy==1.16.1',
```

Comment:
This seems problematic, prefer using `>=` if features from a certain version are required. Otherwise you would trigger a reinstall with a later version of Numpy.

Reply:
Ok, apparently we have a problem caused by TF with Numpy 1.17. This change is necessary for the time being.